assets/js/blocks/cart-checkout/cart/edit.js

```diff
@@ -18,7 +20,55 @@ import EmptyCart from './empty-cart';
 /**
  * Component to handle edit mode of "Cart Block".
  */
-const CartEditor = ( { className } ) => {
+const CartEditor = ( { className, attributes, setAttributes } ) => {
+	const { isShippingCalculatorEnabled, isShippingCostHidden } = attributes;
+
+	const BlockSettings = () => (
+		<InspectorControls>
+			<PanelBody
+				title={ __(
+					'Shipping calculations',
+					'woo-gutenberg-products-block'
+				) }
+			>
+				<ToggleControl
+					label={ __(
+						'Shipping calculator',
+						'woo-gutenberg-products-block'
+					) }
+					help={ __(
+						'Allow customers to estimate shipping.',
+						'woo-gutenberg-products-block'
+					) }
+					checked={ isShippingCalculatorEnabled }
+					onChange={ () =>
+						setAttributes( {
+							isShippingCalculatorEnabled: ! isShippingCalculatorEnabled,
+						} )
+					}
+				/>
+				{ isShippingCalculatorEnabled && (
```

Comment:
This option should be independent of the calculator option, so you can remove the above check.